inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -657,6 +661,17 @@ void GNAPlugin::AddDebugProperties(const InferenceEngine::CNNLayerPtr layer,
 }
 #endif
 
+static bool HasConvolutions(std::shared_ptr<ngraph::Function> f) {
+    /* Currently this transformation is supported only for networks without convolutions.
+     */
+    for (auto& node : f->get_ordered_ops()) {
+        if (std::dynamic_pointer_cast<ngraph::opset7::Convolution>(node) != nullptr) {
+            return true;
+        }
+    }
+    return false;
+}
+
```

Comment:
You can use https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/include/transformations/utils/utils.hpp#L40